### PR TITLE
[bashible] Fix bashible service checking

### PR DESCRIPTION
**Pull request by Diaderius**
Signed-off-by: Konstantin Maltsev <konstantin.maltsev@flant.com>

## Description
There is a fix for bashible service checking regarding to node bootstrap.

## Why do we need it, and what problem does it solve?
It's a hot-fix regarding possible **"Unknown"** output string during bashible checking on other distros.

## What is the expected result?
If service is **active** - there is no needs to perform bootstrap. Other outputs should calls script execution.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: node-manager
type: fix
summary: fix bashible service checking
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Diaderius. I've started the workflow for label 'e2e/run/aws'...

File: modules/040-node-manager/templates/node-group/_static_or_hybrid_script.tpl [hunk 1]
```diff
@@ -91,7 +91,7 @@ touch /var/lib/bashible/first_run
 {{- end }}
 
 checkBashible=$(systemctl is-active bashible.timer)
-if [[ "$checkBashible" == "inactive" ]]; then
+if [[ ! "$checkBashible" == "active" ]]; then
```
Comment (yalosev) [inline modules/040-node-manager/templates/node-group/_static_or_hybrid_script.tpl:94]: why not `[[ "$checkBashible" != "active" ]]` ?

Comment (Diaderius) [reply modules/040-node-manager/templates/node-group/_static_or_hybrid_script.tpl]: @yalosev, ok, fixed, they are functionally equivalent, but your suggestion looks simpler.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Diaderius. I've started the workflow for label 'e2e/run/aws'...

**Review by name212 — APPROVED** [review]

**Review by yalosev — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [3678](https://github.com/deckhouse/deckhouse/pull/3678) to the branch [release-1.43](https://github.com/deckhouse/deckhouse/tree/release-1.43) successful!

**Pull request merged**